.circleci/config.yml

```diff
@@ -66,6 +66,22 @@ commands:
                 key: windows-go-<< parameters.cache_version >>-{{ checksum "go.sum" }}
             - run: 'sh ./scripts/installgo_windows.sh'
             - run: choco install mingw
+      - run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.2
+      - when:
+          condition:
+            equal: [ linux, << parameters.os >> ]
+          steps:
+            - run: GOGC=20 /go/bin/golangci-lint run
+      - when:
+          condition:
+            equal: [ darwin, << parameters.os >> ]
+          steps:
+            - run: $HOME/go/bin/golangci-lint run
+      - when:
+          condition:
+            equal: [ windows, << parameters.os >> ]
+          steps:
+            - run: $HOME/go/bin/golangci-lint.exe run
```

Comment:
> As far as verbose, we do not have that set in our `make lint` command either. Is there an advantage to all the additional messages that aids in debugging for the user? Otherwise it seems like noise that a user has to ignore.

That's right it wasn't set for `make lint` because we didn't want to produce too much noise for user.
But we had it explicitly enabled in `.github/workflows/golangci-lint.yml` to have more detailed info when run on CI.

Reply:
ok I'll update to add verbose shortly!